gcovr/templates/source_page.html

```diff
@@ -49,6 +49,17 @@
 {% endblock %}
 
 {% block content %}
+{% if function_names %}
+<details class="listOfFunctions">
+  <summary>List of functions</summary>
+  <ul>
+    {% for function_name in function_names %}
+    <li><a href="#{{ function_name.1 }}">{{ function_name.0 }}, line {{ function_name.1 }}</a></li>
```

Comment:
The link doesn't seem to work, it seems the line anchors are prefixed with a lowercase-L.

```suggestion
    <li><a href="#l{{ function_name.1 }}">{{ function_name.0 }}, line {{ function_name.1 }}</a></li>
```